frontend/workflows/projectCatalog/src/catalog/index.tsx

```diff
@@ -20,6 +30,15 @@ const initialState: CatalogState = {
   error: undefined,
 };
 
+const Placeholder = () => (
+  <Paper>
+    <div style={{ margin: "32px", textAlign: "center" }}>
+      <Typography variant="h5">There is nothing to display here</Typography>
+      <Typography variant="body3">Please enter a namespace and clientset to proceed.</Typography>
```

Comment:
Should this say 'project name' to proceed? Currently the wording seems a little confusing since we dont search via namespace or anything correct?

Reply:
ah yes, good catch